### ISSUE TITLE
.hack//G.U. Last Recode .css

### ISSUE THREAD

**Issue by makc-ar** [zeroKilo/CCSFileExplorerWV#1]
How about this http://store.steampowered.com/app/525480/hackGU_Last_Recode game?
Example  .css: https://yadi.sk/d/ITGAq_4Z3PR7it

**Comment by zeroKilo** [comment]
yeah, what about it? I have no intention to get back into that community ever again, I got yelled at enough for providing free tools, no thanks

**Comment by makc-ar** [comment]
How much? You have PayPal?

**Comment by zeroKilo** [comment]
nothing, nope and no thanks

**Comment by LightRO** [comment]
Oh that's a pity, especially since you're the only one who knows about the matter. I was hoping that you would keep the project solid and up-to-date. The meshes of G.U would have helped me a lot for a small project ^ ^. But well, you can not do anything anymore.

**Comment by NarutoClassics** [comment]
YOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
